CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## Release History
+### 1.13.0-beta.1 (Unreleased)
+#### Other Changes
+* Fixed an issue where source connector can be stuck in an infinite loop when task got cancelled. [PR 545](https://github.com/microsoft/kafka-connect-cosmosdb/pull/545) 
```

Comment:
There were some CVE issues as well, that we fixed, please check the last couple of PRs and we should mention those changes here. 
Also, shall we update the azure-cosmos version?

Reply:
yea I am thinking to make those changes in the release PR